Changelog 130 (#92)
intellij-rust/intellij-rust.github.io: _posts/2020-09-04-changelog-130.markdown

improved `New project` dialog to support `cargo-generate` templates. In this release we add
[@eupn's template](https://github.com/eupn/rust-procmacro-quickstart-template) that makes it
easy to create a procedural macro project.

<!-- https://github.com/intellij-rust/intellij-rust/pull/5846 -->
* Command completion for `wasm-pack` Run Configurations (that was added in the [previous release](/2020/08/24/changelog-129.html))
and `Run Anything` (Double <kbd>Ctrl</kbd>)

## Performance Improvements

<!-- https://github.com/intellij-rust/intellij-rust/pull/5989 -->
<!-- https://github.com/intellij-rust/intellij-rust/pull/6012 -->
* Don't expand macros disabled by `#[cfg]` attributes. This reduces overall macro expansion time

## Fixes

<!-- https://github.com/intellij-rust/intellij-rust/pull/5949 -->
* Don't unescape backslashes in Rust test output. Fixes many issues with incorrect test output

<!-- https://github.com/intellij-rust/intellij-rust/pull/6028 -->
* Show "missing toolchain" notification in `Cargo.toml` if Rust toolchain is not configured.
Previously this notification was shown only in Rust `.rs` files

<!-- https://github.com/intellij-rust/intellij-rust/pull/6031 -->

> * Fix [bug](https://github.com/intellij-rust/intellij-rust/issues/5961) in name resolution:
> `crate`-relative paths re-exports could be resolved incorrectly

Reviewer: ```suggestion
* Properly resolve re-exports of `crate`-relative paths (see the corresponding [bug](https://github.com/intellij-rust/intellij-rust/issues/5961))
```
